content/doc/developer/internationalization/internationalization-and-localization.adoc

```diff
@@ -0,0 +1,320 @@
+---
+title: Internationalization and Localization
+layout: developersection
+---
+
+https://www.jenkins.io/doc/book/using/using-local-language/[How
+to view Jenkins in your language].
+
+'''''
+
+== Translations to Specific languages
+
+* https://wiki.jenkins.io/JENKINS/65670018[Chinese
+(Traditional) Translation]
+* https://wiki.jenkins.io/JENKINS/135469573[Chinese
+(Simplified) Translation]
+* https://wiki.jenkins.io/JENKINS/French-Localization[French
+Localization]
+* https://wiki.jenkins.io/display/JA/Japanese-Localization[Japanese
+Localization]
+* https://wiki.jenkins.io/JENKINS/Polish-Localization[Polish
+Localization]
+* https://wiki.jenkins.io/JENKINS/Spanish-Translation[Spanish
+Translation]
+* https://wiki.jenkins.io/JENKINS/Brazilian-Portuguese-Translation[Brazilian
+Portuguese Translation]
+
+== What developers need to know/do
+
+Developers must do the following to localize a plugin or other module:
+
+. Generate type-safe `+Messages+` classes from `+Messages.properties.+`.
+. Mark messages in jelly files.
+. Create html files for help tips.
```

Comment:
We have pages that describe these steps.  Let's link to those pages so that the reader can learn more about how to perform that task.
```suggestion
. link:/doc/developer/internationalization/i18n-source-code/[Internationalize messages in Java source code]
. link:/doc/developer/internationalization/i18n-jelly-views/[Internationalize messages in Jelly views]
. (Optional) link:[/doc/developer/crowdin/translating-plugins/Enable Crowdin translation support for the plugin]
```

Reply:
Fixed.